packages/flame/lib/src/components/input/joystick_component.dart

```diff
@@ -72,28 +73,30 @@ class JoystickComponent extends HudMarginComponent with Draggable {
   @override
   Future<void> onLoad() async {
     await super.onLoad();
-    knob.anchor = Anchor.center;
-    knob.position.add(size / 2);
-    _baseKnobPosition = knob.position.clone();
+    knob?.anchor = Anchor.center;
+    knob?.position.add(size / 2);
+    _baseKnobPosition = knob?.position.clone() ?? Vector2.all(0);
```

Comment:
```suggestion
    _baseKnobPosition = knob?.position.clone() ?? Vector2.zero();
```

Reply:
After changing knob back to just `late` this change is gone.